org/gen.go

```diff
@@ -114,17 +113,16 @@ func strPointer(s string) *string {
 
 func convertConfig(cfg Organization) org.FullConfig {
 	allMembers := cfg.Members[:]
-	allMembers = append(allMembers, cfg.Developers...)
 	sort.Slice(allMembers, func(i, j int) bool { return strings.ToLower(allMembers[i]) < strings.ToLower(allMembers[j]) })
 
-	// Insert the developers team, which is handled separately
+	// Insert the members team, which is handled separately
 	closed := org.Closed
-	cfg.Teams["Developers"] = org.Team{
+	cfg.Teams["Members"] = org.Team{
```

Comment:
is creating a new team right? Does this new team have the correct permissions? Won't everyone get notified that they were removed from "Developers"?

Reply:
This makes the permissions the same between members and developers. Yes, it does mean that everyone in developers will get an email that they were removed from developers and added to members. Originally I was going to change everyone to be a developer, but the requirement from #476 was that the new term to be used was members. It didn't make sense to me to refer to them as members in one place and developers in another.

Reply:
So I guess the question here is what's the term we want to use? Developers? Members? Contributors? 

* If we use a different team than the two prior groups then it avoids confusion of prior understandings. 
* Developers may not make sense for, for example, docs writers and other contributors who don't actually write code.

Reply:
maybe we can just rename the team in the UI, then do this, to avoid churn

Reply:
I like the sound of that

Reply:
Could we disable sending e-mails while we do it?

Reply:
@craigbox Like the sound of what? Renaming? What term do we want to use?

Reply:
renaming the teams in the UI, sorry.  I would like the end result to be "members" for the reason you spelled out above, but I don't mind how we get there, and agree with minimising confusion due to automated e-mails.